cimi-resources/src/com/sixsq/slipstream/ssclj/resources/spec/evidence.cljc

```diff
@@ -0,0 +1,25 @@
+(ns com.sixsq.slipstream.ssclj.resources.spec.evidence
+  (:require
+    [clojure.spec.alpha :as s]
+    [com.sixsq.slipstream.ssclj.resources.spec.common :as cimi-common]
+    [com.sixsq.slipstream.ssclj.resources.spec.core :as cimi-core]  
+    [com.sixsq.slipstream.ssclj.util.spec :as su]))
+
+
+; (s/def :cimi.evidence/credentials (s/coll-of ::cimi-common/resource-link))
+(s/def :cimi.evidence/class ::cimi-core/nonblank-string)
```

Comment:
For new resources, use spec identifiers that are namespaced using the same namespace as the containing file.  Essentially just use `::class` to achieve this.  If you need to reference them else where you can then define an alias, and use the form `::alias/class` (like is done for cimi-core).

Reply:
done